tests/requirements.txt

```diff
@@ -0,0 +1 @@
+pytest==6.2.4
```

Comment:
[fix] Document that tests also need `pxr` to be available in your python environment.

Reply:
Added a note in the readme instructions. [7aaa205](https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/3/commits/7aaa20559292f2fc3e4bbce7efb42d26553c419a)